kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/position/GNSSType.java

```diff
@@ -0,0 +1,36 @@
+package org.eclipse.kura.position;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum GNSSType {
+
+    BEIDOU("Beidou"),
+    GALILEO("Galileo"),
+    GLONASS("Glonass"),
+    GPS("Gps"),
+    OTHER("Other"),
+    UNKNOWN("Unknown");
+
```

Comment:
Should we consider also the regional systems QZSS and IRNSS?

```suggestion
    BEIDOU("Beidou"),
    GALILEO("Galileo"),
    GLONASS("Glonass"),
    GPS("Gps"),
    QZSS("QZSS"),
    IRNSS("IRNSS"),
    OTHER("Other"),
    UNKNOWN("Unknown");

```